# Translation of Odoo Server.
# This file contains the translation of the following modules:
# 	* account_invoice_constraint_chronology
#
msgid ""
msgstr ""
"Project-Id-Version: Odoo Server 16.0\n"
"Report-Msgid-Bugs-To: \n"
"Last-Translator: \n"
"Language-Team: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Plural-Forms: \n"

#. module: account_invoice_constraint_chronology
#: model:ir.model.fields,field_description:account_invoice_constraint_chronology.field_account_journal__check_chronology
msgid "Check Chronology"
msgstr ""

#. module: account_invoice_constraint_chronology
#. odoo-python
#: code:addons/account_invoice_constraint_chronology/model/account_move.py:0
#, python-format
msgid ""
"Chronology conflict: A conflicting draft invoice dated before {date_invoice}"
" exists, please validate it first."
msgstr ""

#. module: account_invoice_constraint_chronology
#. odoo-python
#: code:addons/account_invoice_constraint_chronology/model/account_move.py:0
#, python-format
msgid ""
"Chronology conflict: A conflicting validated invoice dated after "
"{date_invoice} exists."
msgstr ""

#. module: account_invoice_constraint_chronology
#. odoo-python
#: code:addons/account_invoice_constraint_chronology/model/account_move.py:0
#, python-format
msgid ""
"Chronology conflict: An invoice with a higher number {highest_name} dated "
"before {date_invoice} exists."
msgstr ""

#. module: account_invoice_constraint_chronology
#. odoo-python
#: code:addons/account_invoice_constraint_chronology/model/account_move.py:0
#, python-format
msgid ""
"Chronology conflict: Invoice {name} cannot be {time} invoice {inv_name}."
msgstr ""

#. module: account_invoice_constraint_chronology
#: model:ir.model,name:account_invoice_constraint_chronology.model_account_journal
msgid "Journal"
msgstr ""

#. module: account_invoice_constraint_chronology
#: model:ir.model,name:account_invoice_constraint_chronology.model_account_move
msgid "Journal Entry"
msgstr ""
